src/main/resources/templates/generators/overviewTrends.vm

```diff
@@ -5,7 +5,7 @@
   #includeTitle("Trends Overview")
 
   <script type="text/javascript">
-  #parse("/templates/js/trends-chart.js")
+  #parse("/templates/js/trends-chart.js.vm")
```

Comment:
1. You should also rename the other js files where they are used as templates but that is outside the scope of this change. 

2. Hate to do this. But it works for me:

![trends-charts-js vm](https://cloud.githubusercontent.com/assets/6946919/21573453/bfb3de58-cee4-11e6-94d9-4ba723ecf09c.png)